docs/quickstart/rbac.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Role-Based Access Control (RBAC)
+hide_table_of_contents: false
+---
+
+# RBAC
+
+In order to grant permissions to users, we integrated a Role-Based Access Control (RBAC) system. We defined 4
+roles: `admin`, `project_admin`, `approver` and `member`.
+The user creating an organization is automatically assigned the `admin` role.
+Every time a user is [invited](/cli-reference/strm/invite/users.md) to your organization, the `member` role is assigned.
+Roles can be managed via the [`manage` command in the cli](/cli-reference/strm/manage/user-roles.md).
+
+## Brief overview of permissions per role
```

Comment:
The table is nice, though it actually consists of 2 parts:
1. Permissions + Scope
2. Default roles

I think it would be good to split this into two tables. First, we should explain the base permissions and the scopes they apply to. This can be done with a table.
Next, another table should show which permissions are attached to which roles. In the future, we could also further standardize the role and permission notation (reference to how GCP does it with `roles/storage.objectAdmin` that has several permissions attached).

Reply:
Is this more to your liking?